internal/servers/controller/handlers/targets/target_service.go

```diff
@@ -204,7 +204,7 @@ func (s Service) GetTarget(ctx context.Context, req *pbs.GetTargetRequest) (*pbs
 	if authResults.Error != nil {
 		return nil, authResults.Error
 	}
-	t, ts, err := s.getFromRepo(ctx, req.GetId())
+	t, ts, _, err := s.getFromRepo(ctx, req.GetId())
```

Comment:
@talanknight I returned the credential library up to the service handler, but I didn't do anything from this point on.  I was not sure if I should update the [proto](https://github.com/hashicorp/boundary/blob/main/internal/proto/local/controller/api/resources/targets/v1/target.proto#L22) to include the cred libs or if you would be doing that as part of the API work.

Reply:
I can update that as part of the API work.

Reply:
Sounds good